src/patient-registration/patient-registration.component.tsx

```diff
@@ -503,6 +503,17 @@ export const PatientRegistration: React.FC = () => {
           results.then(response => {}).catch(err => {});
 
           navigate({ to: getAfterUrl(response.data.uuid) });
+          existingPatient
+            ? showToast({
+                description: t("The patient's information has been successfully updated"),
```

Comment:
Can we get proper locale keys such as `updatePatientSuccessDescription`, `updatePatientSuccessTitle` etc.